src/EditorFeatures/Test/EditAndContinue/Helpers/MockDiagnosticAnalyzerService.cs

```diff
@@ -44,6 +44,9 @@ public Task<ImmutableArray<DiagnosticData>> GetDiagnosticsForIdsAsync(Solution s
         public Task<ImmutableArray<DiagnosticData>> GetDiagnosticsForSpanAsync(Document document, TextSpan? range, string? diagnosticId = null, bool includeSuppressedDiagnostics = false, CodeActionRequestPriority priority = CodeActionRequestPriority.None, Func<string, IDisposable?>? addOperationScope = null, CancellationToken cancellationToken = default)
             => throw new NotImplementedException();
```

Comment:
should this call just because an extension method that defers to the new one?  i'd prefer interfaces not have methods that are effectively overloads to avoid passing an arg.

Reply:
That sounds reasonable, I was concerned if some partner is still using this API via IVT, but just converting it to an extension method seems fine.